main.go

```diff
@@ -133,7 +134,10 @@ func findNums(prefix string, kvs map[string]interface{}, out io.Writer) {
 			el.Println("Error determining hostname", err)
 		}
 
-		_, err = fmt.Fprintf(out, hostname+".rsyslog.%v.%v:%d|g\n", prefix, sanitizeKey(k), int(vf))
+		// Replace periods with hyphens for graphite compatbility
+		hostnameFormatted := strings.Replace(hostname, ".", "-", -1)
```

Comment:
NAB: You can use `ReplaceAll` and then skip the -1

```suggestion
		hostnameFormatted := strings.ReplaceAll(hostname, ".", "-")
```